frame/dex/rpc/src/lib.rs

```diff
@@ -70,7 +70,6 @@ impl From<Error> for i32 {
 	}
 }
 
-#[async_trait]
```

Comment:
This was speculative. Do we need it here?

Reply:
I wish I know :D

Reply:
ok then we remove it until something breaks.